admin/schroot/Makefile

```diff
@@ -1,23 +1,25 @@
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=schroot
-PKG_VERSION:=1.6.10
-PKG_RELEASE:=3
+PKG_NAME:=reschroot
+PKG_VERSION:=1.6.13
+PKG_RELEASE:=1
 
-PKG_SOURCE:=$(PKG_NAME)_$(PKG_VERSION).orig.tar.xz
-PKG_SOURCE_URL:=http://deb.debian.org/debian/pool/main/s/schroot
-PKG_HASH:=3ce8dfd9cb97b099e4b6d4ccec421d6cc8c9ef84574681e928a12badb5643d0b
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeberg.org/shelter/reschroot/archive/release
```

Comment:
Uhh, at first glance, it looks like this repository and the other one on Debian has the same recent commits of the same author, but when looking here https://salsa.debian.org/debian/schroot/-/commit/37f5af211fedc939593f6dbb2992f9d8bbaa866b he is referring to codeberg, thus I it is alright.